src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.color.ColorSuplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    
+    public Figure getRandomFigure() {
+        return getFigure();
+    }
+
+    private Figure getFigure() {
+        final int figures = 5;
+        final int maxSize = 99;
```

Comment:
these should be private static final constants (class-level)

Reply:
You already have these constants